test/SILOptimizer/diagnostic_constant_propagation.swift

```diff
@@ -1,342 +1,340 @@
-// FIXME(integer): with new integer protocols implemented the overflows are no
-// longer caught: <rdar://problem/29937936>
-// XFAIL: *
+// These are tests for diagnostics produced by constant propagation pass.
+// Due to the change in the implementation of Integer initializers some of the tests here that must fail don't currently.
+// Such tests have comments describing the desirable behavior. They are false negatives now but have to be addressed in the future.
+// References: <rdar://problem/29937936>, <rdar://problem/29939484>, <https://bugs.swift.org/browse/SR-5964>, <rdar://problem/39120081>
 
 // RUN: %target-swift-frontend -emit-sil -primary-file %s -o /dev/null -verify
 
 // REQUIRES: PTRSIZE=64
+// <rdar://problem/19508336> Extend test/SILOptimizer/diagnostic_constant_propagation.swift to 32-bit platforms
```

Comment:
We should probably leave the FIXME in place here since we haven't added 32-bit tests yet.